src/content/docs/style-guide/quick-reference/capitalization.mdx

```diff
@@ -7,7 +7,7 @@ redirects:
   - /docs/new-relic-only/basic-style-guide/writing-guidelines/capitalization
 ---
 
-In general, we only capitalize things when we need to. Read on for some guidelines on how to decide what to capitalize in a document's title, headings, products, features, and other elements of the page.
+In general, we only capitalize things when we need to. Excess capitalization is distracting to our readers and restricts accessibility to our documents. Read on for some guidelines on how to decide what to capitalize in a document's title, headings, products, features, and other elements of the page.
```

Comment:
I'd recommend considering some simpler language you might use here. :) I'm happy to provide a suggestion, but I'll let you take a crack at it first.

Reply:
@paperclypse I like the coordinated balance so I was thinking: "Redundant capitalization is distracting to our readers and limits accessibility to our documents." What are your thoughts?

Reply:
Instead of redundant or excess, I'd go with "Over use of capitalization is..." :)

And you may want to make it clearer (I had to think about it for a few minutes) that screen readers don't indicate whether a word is capitalized or not, which limits accessibility for people with a visual impairment.